### PR TITLE
Add docs link

### LINKED ISSUE #28: Add link to docs
Add a link to https://vega.github.io/editor-backend/ to https://vega.now.sh/

### PR DESCRIPTION
**Pull request by RisingGeek**
Fixes #28

### CODE REVIEW

**Comment by RisingGeek** [comment]
How can I authorize the now deployment?

**Comment by domoritz** [comment]
It’s fine. We don’t need the authorization for pull requests. Thanks for your PR.

**Comment by RisingGeek** [comment]
Oh, okay. @domoritz please review the pull request.

**Comment by domoritz** [comment]
@algomaster99 will review and merge

File: src/urls.ts [hunk 1]
```diff
@@ -27,6 +27,10 @@ export const authUrl: any = {
    * Endpoint to verify if the user is authenticated.
    */
   isAuthenticated: `${authMain}/check`,
+  /**
+   * URL for Vega Editor API documentation
+   */
+  docs: 'https://vega.github.io/editor-backend',
```
Comment (algomaster99) [inline src/urls.ts]: Create it as a separate variable. Not under `authUrl`. Make it as `docsUrl` at the bottom of the file.

**Comment by algomaster99** [comment]
@RisingGeek Also, please make sure your commit messages start with a capital letter. The starting word should be a verb in the present tense. For example: `Add link for editor-backend documentation`

File: src/urls.ts (hunk 1 above)
Comment (RisingGeek) [reply src/urls.ts]: Okay

**Pull request merged**